src/tic_tac_toe/game.clj

```diff
@@ -40,10 +40,10 @@
          :else (game-loop marked-board (switch-marker marker)))))
 
 (defn game-loop-human-computer  [board marker]
-  (cond  (is-tie? board)  (println "Game Over - Tie.")
-         (is-win? board)  (println "Game Over - Computer Wins.")
+  (cond  (is-tie? board) (println "Game Over - Tie.")
+         (is-win? board) (println "Game Over - Computer Wins.")
          :else
   (let  [marked-board  (human-turn board "X")]
-   (cond  (is-win? marked-board)  (println "Game Over - Huamn Wins.")
-         (is-tie? marked-board)  (println "Game Over - Tie.")
-         :else  (do  (game-loop-human-computer  (computer-turn marked-board "O") "O" ))))))
+   (cond (is-win? marked-board) (println "Game Over - Huamn Wins.")
+         (is-tie? marked-board) (println "Game Over - Tie.")
+         :else  (do (game-loop-human-computer (computer-turn marked-board "O") "O" ))))))
```

Comment:
Rather than using strings for X and O, I think you could use symbols 'X 'O throughout